sphinxcontrib/video.py

```diff
@@ -35,22 +39,34 @@ def run(self):
         height = get_option(self.options, "height", "")
         autoplay = get_option(self.options, "autoplay", False)
         nocontrols = get_option(self.options, "nocontrols", False)
+        loop = get_option(self.options, "loop", False)
+        muted = get_option(self.options, "muted", False)
+        additionalsource = get_option(self.options, "additionalsource", "")
         
         return [video(
             path=self.arguments[0],
             alt=alt, 
             width=width,
             height=height, 
             autoplay=autoplay, 
-            nocontrols=nocontrols
+            nocontrols=nocontrols, 
+            loop=loop, 
+            muted=muted, 
+            additionalsource=additionalsource
             )]
 
 def visit_video_node(self, node):
     extension = os.path.splitext(node["path"])[1][1:]
 
+    logger = logging.getLogger(__name__)
+    if node["additionalsource"] == "": 
+        logger = logging.getLogger(__name__)
+        logger.warning('Video tag is missing additional source. This could break compatibility with some browsers.', location=node)
```

Comment:
There should probably be a way to opt out of this warning (if there's not one at the Sphinx level already). The main browsers lacking H.264 support are on Linux distributions which refuse to include an H.264 decoder for patent reasons.

Reply:
That might be true. In 2020 you can drop h.264 support altogether. Only Safari and Internet Explorer (which is in its end of life phase), don't support VP9. All in all, it's a pretty minor portion of all browser users.

Jokes aside, in our project it made absolute sense to have this warning, as we cannot include proprietary codecs for licensing reasons, but we do support macOS. So basically we use VP9 and Safari gets its h.264.

I do agree that if you don't care about macOS support you can just keep with VP9, while if you don't care about open formats, you can just keep with h.264.

Not sure how/if Sphinx's way of suppressing warnings integrates into this, but it would make sense to put a little documentation on the readme, just in case. https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-suppress_warnings

Reply:
> In 2020 you can drop h.264 support altogether.

I said dropping WebM, not H.264 :slightly_smiling_face: